LoopBackTests/LBPersistedModelDataTypesTests.m

```diff
@@ -0,0 +1,232 @@
+//
+//  LBPersistedModelDataTypesTests.m
+//  LoopBack
+//
+//  Created by hideya kawahara on 1/6/16.
+//  Copyright © 2016 StrongLoop. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "LBPersistedModel.h"
+#import "LBRESTAdapter.h"
+#import "SLRemotingTestsUtils.h"
+
+
+@interface Widget2 : LBPersistedModel
```

Comment:
Please use a more descriptive name. What is the difference between `Widget` and `Widget2`?